src/metric/event.rs

```diff
@@ -1,6 +1,13 @@
 use metric::{LogLine, Telemetry};
 use std::sync;
 
+/// Supported event encodings.
+#[derive(Debug, PartialEq, Serialize, Deserialize, Clone)]
+pub enum Encoding {
```

Comment:
Can we also ship with an Untyped encoding?

Reply:
Added Encoding::None.

Reply:
I decided to go with Encoding::Raw.